src/kartikey/WebElements_HtmlPage.java

```diff
@@ -0,0 +1,95 @@
+package kartikey;
+
+import java.util.Scanner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebElements_HtmlPage {
+	boolean verifyTitle(WebDriver driver, String title) {
+		if (driver.getTitle().equals(title))
+			return true;
+		else
+			return false;
+	}
+
+	void fillform(WebDriver driver, String url) {
+		driver.get(url);
+		driver.findElement(By.id("first name")).sendKeys("Kartikey");
+		driver.findElement(By.id("last name")).sendKeys("Dev");
+		driver.findElement(By.id("E-mail")).sendKeys("Kd@gamil.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Capgemini");
+		if (driver.findElement(By.id("maleG")).isSelected() == true)
```

Comment:
You made wrong validation, for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.